### ISSUE TITLE
[BUG] Press the key to delay Re-upload the video

### ISSUE THREAD

**Issue by bsjssjjs** [PojavLauncherTeam/PojavLauncher#4711]
### Describe the bug

https://github.com/PojavLauncherTeam/PojavLauncher/assets/136832400/595993ea-733a-4aae-9d90-3c521ec507b7

### The log file and images/videos

https://github.com/PojavLauncherTeam/PojavLauncher/assets/136832400/dc5c8ba1-fc13-489c-85ad-4a7f8ce89464

When playing version 1.20.2, the key delay suddenly became very large

### Steps To Reproduce

```markdown
1.Start the game
2.Enter the game
3.Press the key
```

### Expected Behavior

A similar problem does not occur in version 1.20.2

### Platform

```markdown
It was reported to me by someone else, so it's not clear
- Device model:
- CPU architecture: 
- Android version: 
- PojavLauncher version:
```

**Comment by GaucovyAero** [comment]
![Screenshot_20231007-181726_Fennec](https://github.com/PojavLauncherTeam/PojavLauncher/assets/85581164/90947ad5-27ca-4b02-be19-3dc6f0115814)

your videos are seriously borked man

**Comment by MovTery** [comment]
> ![Screenshot_20231007-181726_Fennec](https://github.com/PojavLauncherTeam/PojavLauncher/assets/85581164/90947ad5-27ca-4b02-be19-3dc6f0115814)
> 
> your videos are seriously borked man

I think this is a problem with your video decoding
https://github.com/PojavLauncherTeam/PojavLauncher/assets/123115192/839f885e-7727-4f56-a6ef-4a3ab1af3bce

**Comment by GaucovyAero** [comment]
on my pc it says this lol, please record with an encoder that actually works on something else than your phone

![image](https://github.com/PojavLauncherTeam/PojavLauncher/assets/85581164/e2903c49-5d08-4028-add7-d7287d95b146)

**Comment by GaucovyAero** [comment]
oh, i downloaded the video and that worked (with mpv on PC), be sure to have `mixin.core.render.world=false` in your sodium mixins with patched sodium 0.5+

**Comment by GaucovyAero** [comment]
seems like the issue with your videos is that you are using H265/HEVC to record them, which results in most browsers not being able to decode them (it should be listed as less space consuming format or more effective or something in your screen recorder settings, disable that to use normal h264)
